filter.imp

```diff
@@ -1,13 +1,111 @@
 [
-   { include: ["@.asm-generic/unaligned.h.", private, "<asm/unaligned.h>", "public"] },
+   # The following asm-generic files will always be replaced by asm files
+   # This list was generated from include/uapi/asm-generic/Kbuild
+   { include: ["@.asm-generic/auxvec.h.", private, "<asm/auxvec.h>", "public"] },
+   # bit twiddling operations should use linux/bits.h
+   { include: ["@.asm-generic/bitsperlong.h.", private, "<linux/bits.h>", "public"] },
```

Comment:
I guess if a .c file is only referencing `__BITS_PER_LONG`, we should not be pulling in all of linux/bits.h. Drop this comment and replace the public file with asm/bitsperlong.h?

Or perhaps this should be 2 mappings.

asm-generic/bitsperlong.h -> asm/bitsperlong.h (in this section)
asm/bitsperlong.h -> bits.h (in the other section further down)